### ISSUE TITLE
SwitchBot Keypad scenario request

### ISSUE THREAD

**Issue by frean087** [AdyRock/com.switchbot#91]
Would be more then fantastic if there was a possibility to get the follow in automation to work:

When correct code, nfc or fingerprint is registered by the SwitchBot Key pad. 
Then activate scene...

Or add have the keypad as an isolated unit in homey where you can have some deferent scenarios built.

Thanks in advance!

**Comment by AdyRock** [comment]
I agree that would be great. But the API doesn't provide any feedback from the keypad. There is an outstanding request with Switchbot to provide more information from the keypad but it has not been answered. 
If you search for Switchbot open API  in Google it should find their GitHub repo and the request is in the issues. Please add a comment their to see if they will do something.
